neo/Wallets/Wallet.cs

```diff
@@ -224,6 +224,7 @@ public virtual WalletAccount Import(string nep2, string passphrase)
         {
             if (tx.Outputs == null) tx.Outputs = new TransactionOutput[0];
             if (tx.Attributes == null) tx.Attributes = new TransactionAttribute[0];
+            if (fee < ProtocolSettings.Default.MinimumNetworkFee) fee = ProtocolSettings.Default.MinimumNetworkFee;
```

Comment:
This should be true only if `tx` is `InvocationTransaction`.

Reply:
Have fixed